### ISSUE TITLE
Some things about the way the download presentations button work (UX)

### ISSUE THREAD

**Issue by JuanCasad0** [bigbluebutton/bigbluebutton#16529]
Hi!
When testing the new beta6 I noticed two things that I personally think could be improved when it comes to get a better user experience. Just as a suggestion:

In bbb.2.5, a teacher can set as downloadable any current presentation. Students can see a little arrow, and the download becomes transparent. This has changed in 2.6: now the teacher must click in "send to chat" button. This starts converting the current pdf, and takes some time before the download is notified in the chat area or becomes available.

However... there can be moments when a teacher needs the student to download the pdf before he/she annotates anything (at the beginning of the session, for instance). So, in this case, the teacher must click in the "send to chat" button, wait for the conversion to be done, wait until the students download the file... So I think that when it comes to this, BBB2.5 handled this situation in a better way.

Another thing I've found that could improve the user experience is the fact that when the file is sent to the chat, that link doesn't show on the recording, so I think there's margin to improve it here, just like when a Youtube URL is shared live on the session, and it appears later on the chat recording.

So, in summary:
1) Could it be a way to  have back the download button for the current presentation without having it converted?
2) Could we have in the recording the URL to download the presentation that was sent to the chat once the teacher, before ending the session, allows the students to download the annotated pdf?

Thanks a lot!

**Comment by danielpetri1** [comment]
Hi! Since PR #16421 the conversion is skipped if no annotations are available in the presentation, making it instantly available in chat.

**Comment by JuanCasad0** [comment]
Good to know @danielpetri1! Thanks for the info. When I tested this, the conversion took place. 

It'd be interesting to know if my request #2 could be possible!

Thanks again!
